### PR TITLE
fix(playground): simplify default messages

### LINKED ISSUE #5533: [playground] improve default playground messages
Screenshot 2024-11-26 at 9 57 32 AM

Arize for comparison

### PR DESCRIPTION
**Pull request by axiomofjoy**
Screenshot 2024-11-26 at 5 18 42 PM

- I find myself frequently deleting the default system prompt when testing out playground
- Gemini requires non-empty message content, so you can't just hit run with `{{question}}` and an empty template variable
- Seeing for the first time, I think it helps to have other content aside from the template variable in the prompt to grok the connection between template variables in the prompt and values in the right panel.

resolves #5533

### CODE REVIEW

File: app/src/pages/playground/__tests__/playgroundUtils.test.ts [hunk 1]
```diff
@@ -98,12 +98,12 @@ const defaultTemplate = {
     {
       id: 0,
       role: "system",
-      content: "You are a chatbot",
+      content: "Welcome to Phoenix Playground!\n\nYou are a helpful assistant.",
```
Comment (mikeldking) [inline app/src/pages/playground/__tests__/playgroundUtils.test.ts:101]: Not sure how this helps the user - it's just more text to have to delete

File: app/src/pages/playground/__tests__/playgroundUtils.test.ts [hunk 2]
```diff
@@ -98,12 +98,12 @@ const defaultTemplate = {
...
     },
     {
       id: 1,
       role: "user",
-      content: "{{question}}",
+      content: "Answer the question: {{question}}",
```
Comment (mikeldking) [inline app/src/pages/playground/__tests__/playgroundUtils.test.ts:106]: nit: similar to above

File: app/src/pages/playground/__tests__/playgroundUtils.test.ts (hunk 1 above)
Comment (mikeldking) [reply app/src/pages/playground/__tests__/playgroundUtils.test.ts:101]: Also "Welcome to Phoenix Playground!" shouldn't be a part of the system prompt?

Comment (axiomofjoy) [reply app/src/pages/playground/__tests__/playgroundUtils.test.ts:101]: It's not necessary, can close.

**Comment by axiomofjoy** [comment]
- I find myself frequently deleting the default system prompt when testing out playground
- Gemini requires non-empty message content, so you can't just hit run with `{{question}}` and an empty template variable
- Seeing for the first time, I think it helps to have other content aside from the template variable in the prompt to grok the connection between template variables in the prompt and values in the right panel.

File: app/src/pages/playground/__tests__/playgroundUtils.test.ts [hunk 3]
```diff
@@ -82,28 +82,23 @@ const expectedPlaygroundInstanceWithIO: PlaygroundInstance = {
...
     {
       id: 0,
-      role: "system",
-      content: "You are a chatbot",
-    },
-    {
-      id: 1,
       role: "user",
-      content: "{{question}}",
+      content: `How many r's in "{{fruit}}"?`,
```
Comment (mikeldking) [inline app/src/pages/playground/__tests__/playgroundUtils.test.ts:101]: Can we leave this to be answer the question? I find the question variable to be much more generic and useful when switching to dataset mode

File: app/src/store/playground/playgroundStore.tsx [hunk 4]
```diff
@@ -83,22 +83,17 @@ export const _resetToolId = () => {
...
-    {
-      id: generateMessageId(),
-      role: "system",
-      content: "You are a chatbot",
-    },
     {
       id: generateMessageId(),
       role: "user",
-      content: "{{question}}",
+      content: `How many r's in "{{fruit}}"?`,
```
Comment (mikeldking) [inline app/src/store/playground/playgroundStore.tsx:89]: Let's move this into the question variable. Seems more scalable that way

File: app/src/store/playground/playgroundStore.tsx [hunk 5]
```diff
@@ -175,7 +170,9 @@ export const createPlaygroundStore = (initialProps: InitialPlaygroundState) => {
       // variablesValueCache is used to store the values of variables for the
       // manual input mode. It is indexed by the variable key. It keeps old
       // values when variables are removed or when switching to dataset input so that they can be restored.
-      variablesValueCache: {},
+      variablesValueCache: {
+        fruit: "strawberry",
```
Comment (mikeldking) [inline app/src/store/playground/playgroundStore.tsx:174]: Let's make this be question

File: app/src/store/playground/playgroundStore.tsx [hunk 6]
```diff
@@ -83,22 +83,17 @@ export const _resetToolId = () => {
 const generateChatCompletionTemplate = (): PlaygroundChatTemplate => ({
   __type: "chat",
   messages: [
-    {
```
Comment (mikeldking) [inline app/src/store/playground/playgroundStore.tsx:86]: This is the default system prompt for some other frameworks. I tend to find having the slot available to edit to be a nice default...

**Pull request closed without merging**